apps/patient/src/views/Status.tsx

```diff
@@ -215,10 +206,10 @@ export const Status = () => {
               ) : null}
             </Box>
           </VStack>
-          {enrichedPharmacy ? (
+          {pharmacy ? (
             <Box width="full">
               <PharmacyCard
-                pharmacy={enrichedPharmacy}
+                pharmacy={pharmacyWithHours}
```

Comment:
I see this was changed from `enrichedPharmacy`, but in `Pharmacy.tsx` I see you're using the type with that name too... is a pharmacy with hours = enriched?

Reply:
yeah